### ISSUE TITLE
UIWebView deprecated

### ISSUE THREAD

**Issue by shenck** [Iterable/swift-sdk#156]
As the title states.  Apple will start rejecting apps still using this. Below is the email I received.

```Dear Developer,

We identified one or more issues with a recent delivery for your app, ________ . Your delivery was successful, but you may wish to correct the following issues in your next delivery:

ITMS-90809: Deprecated API Usage - Apple will stop accepting submissions of apps that use UIWebView APIs . See https://developer.apple.com/documentation/uikit/uiwebview for more information.

After you’ve corrected the issues, you can use Xcode or Application Loader to upload a new binary to App Store Connect.

Best regards,

The App Store Team

**Comment by tapashmajumder** [comment]
Thanks for brining this to our notice.

**Comment by damirstuhec** [comment]
@tapashmajumder what is the timeline of fixing this in your SDK?

**Comment by tapashmajumder** [comment]
Very soon. In a few days.

**Comment by vrutberg** [comment]
Has there been any development on this?

**Comment by tapashmajumder** [comment]
Please give us a few more days. We are planning on fixing a couple of issues with the next release.

**Comment by tapashmajumder** [comment]
We just released 6.1.1 with a fix for this.
